source/leuven/wice_quick_start.rst

```diff
@@ -16,54 +16,57 @@ There are several type of nodes in the wICE cluster: normal compute nodes, GPU n
+wICE cluster uses a different workload manager than Genius: Slurm instead of Torque+Moab. More information about converting pbs scripts and commands into Slurm can be found :ref:`here <Antwerp Slurm_convert_from_PBS>`
 
 .. _submit to wice compute node:
 
 Submit to a compute node
 ~~~~~~~~~~~~~~~~~~~~~~~~
-The submission will occur from login nodes of Genius cluster, therefore you need to always remember to specify the cluster you need to use. 
+In these examples, the submission will be done from login nodes of the Genius cluster, therefore you need to always remember to specify the cluster you want to use.
 
 To submit to a compute node you need to provide the required number of nodes and cores. For example to request 2 nodes with each 72 cores for 2 hours you can submit like this::
 
    $ sbatch --cluster=wice --nodes=2 --ntasks-per-node=72 --time=2:00:00  -A lp_myproject  myjobscript.slurm
-  
+
 
 Submit a long job to a compute node
-~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 To submit to a compute node a job longer than 3 days you need to request a separate partition:
 
 ::
 
-   $ sbatch --cluster=wice --nodes=2 --ntasks-per-node=72 --time=6-16:00:00 –-partition=batch_long -A lp_myproject  myjobscript.slurm
+   $ sbatch --cluster=wice --nodes=2 --ntasks-per-node=72 --time=6-16:00:00 --partition=batch_long -A lp_myproject  myjobscript.slurm
 
 
 .. _submit to wice interactive node:
 
 Submit to an interactive partition
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The interactive nodes are located in a separate partition. The users are allowed to request maximum of 8 cores for maximum walltime of 16 hours. In case of these nodes it is important to use then in interactive way (so not submittiong the script) and specyfying the requirements, for example:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The interactive nodes are located in a separate partition. The users are allowed to request a maximum of 8 cores for a maximum walltime of 16 hours. These nodes are intended for interactive use. Instead of submitting a job script, you open an interactive session on a compute node as follows:
 
 ::
 
-   $ srun -n 1 -t 01:00:00 -A lp_myproject --partition=interactive --cluster=wice --pty bash –l
+   $ srun -n 1 -t 01:00:00 -A lp_myproject --partition=interactive --cluster=wice --pty bash -l
 
-If a GPU is necessary for the visualization process - it can be requested (max 1 GPU per max 8 cores and max 16 hours). The available GPU is a single A100, but split in 7 'virtual' GPU's. Additionaly, X11 forwarding should be on:
+If a GPU is necessary for the visualization process - it can be requested (max 1 GPU per max 8 cores and max 16 hours). The available GPU is a single A100, but split in 7 'virtual' GPU's. Additionally, X11 forwarding should be on:
```

Comment:
```suggestion
If a GPU is necessary for the visualization process, it can be requested (max 1 GPU and 8 cores for at most 16 hours). The available GPU is a single A100 which has been split in 7 GPU instances (one of which will be allocated to your job). Additionally, X11 forwarding should be enabled:
```